docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,11 @@
+The native query passed to the underlying data source is required to return a
+table as a result set. This can be a one column table with one row. No
```

Comment:
```suggestion
table as a result set. This can be a one column table with one row. No
```

Why is this sentence required? It makes it sound like other "shapes" aren't ideal.

Reply:
@Jessie212 will remove